src/main/java/edu/asu/commons/foraging/conf/RoundConfiguration.java

```diff
@@ -399,7 +399,7 @@ public boolean isLabDollarsEnabled() {
 
     public String toCurrencyString(double amount) {
         if (isLabDollarsEnabled()) {
-            return String.format("%.2f lab dollars", amount);
+            return String.format("%.2f SEK", amount);
```

Comment:
Since this will also be run in Colombia shouldn't this still be using the generic term `lab dollars`?